### ISSUE TITLE
New Function: audio_sound_pan();

### ISSUE THREAD

**Issue by DarkD04** [YoYoGames/GameMaker-Bugs#6075]
### Is your feature request related to a problem?

Sound emitters tend to be complicated to setup, especially since you wanna do very basic panning.

### Describe the solution you'd like

I wanna requestion much simpler function to use called audio_sound_pan(index, pan); and arguments for this function are index and pan value

index = the sound to be modified.
pan = value for pan, for example -1.0 is to your left ear and 1.0 is to your right ear

### Describe alternatives you've considered

*

### Additional context

*

**Comment by stuckie** [comment]
This is not something we are planning.

**Comment by ShadowTheDragon** [comment]
Why is it "Not something we are planning"?

**Comment by stuckie** [comment]
You could get the same effect with moving the emitter from the listener.

**Comment by ShadowTheDragon** [comment]
Yes it's possible to get it with emitters and a listener, but as this post said listener/emitters are a lot to set up for _just_ audio panning. I'm pretty sure all that we want here is a second option that's more convenient / less complicated to set up and saves a bit of time so we can invest that time elsewhere in our games.

**Comment by KingDevyn** [comment]
Came here to say what everyone else is saying. The 3D audio has other side effects that, at best, require hacky solutions to properly replicate a simple LR volume pan, if at all.

If you're working with stereo sounds, then the manual explicitly warns against using the audio emitter functions to begin with.

**Comment by DarkD04** [comment]
Honestly i think saying "not planned" defeats the whole purpose of having feature requests, I requested for this feature so I dont have to do the world's biggest hack code just to have basic from left to right sound panning

**Comment by SpencerNagg** [comment]
I agree, when I was first learning to use audio in GM I was surprised there was no easy way to do this outside of using emitters. Adjusting L/R panning is a pretty fundamental audio manipulation.

Having to use the Emitter system is less practical when you just want quick and fast panning, say for a short sound effect that bounces from one ear to the other.
